src/Plugin.php

```diff
@@ -86,6 +90,45 @@ public static function upload_mime_types(array $mimes) {
   }
 
   /**
+   * Removes svg width and height attributes from the content.
+   *
+   * @implements the_content
+   */
+  public static function the_content($content) {
+    $content = preg_replace_callback('@<img [^>]*src=.+?.svg[^>]*>@', function ($matches) {
+      return preg_replace('@((?:width|height)=.+?\s+)@', '', $matches[0]);
```

Comment:
1. Yes, also here, why can't it be achieved like it is currently?
2. 👍 Thanks for pointing out! 
3. This is partially possible. For example in SCRM we set the default value for width/height to 128 if it was not present. So those won't be stripped then.